src/dev-tools.js

```diff
@@ -1,8 +1,11 @@
 import remote from 'remote';
 
 export default class DevTools {
-  constructor() {
+  constructor(initial) {
```

Comment:
Thinking this could probably be named better. Or, maybe done outside (e.g., in index). Not 100% sure which was is best. Then again, don't really want to pile on more stuff into index.

`initallyOpen`?